### PR TITLE
Add parquet to changefeed format option

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-6799

Adds documentation for the parquet to the `format` option for changefeeds. 
- Added to core and enterprise changefeed SQL pages.
- Updated the exporting with changefeed page + changefeed messages page.

### CODE REVIEW

**Review by jayshrivastava — APPROVED** [review]
Looks great!

File: src/current/v23.1/changefeed-for.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ Option | Value | Description
 
 #### Avro limitations
 
+Creating a changefeed using Avro is available in Core and [{{ site.data.products.enterprise }} changefeeds](create-changefeed.html) with the [`confluent_schema_registry`](create-changefeed.html#confluent-registry) option.
```
Comment (ashleykwtam) [inline src/current/v23.1/changefeed-for.md:85]: Do we still make mention of Core? Or are we only calling it Self-Hosted now and referring it to `CockroachDB {{ site.data.products.core }}` ?

**Review by ashleykwtam — APPROVED** [review]
LGTM, other than the mentions of core!

File: src/current/v23.1/changefeed-for.md (hunk 1 above)
Comment (kathancox) [reply src/current/v23.1/changefeed-for.md:85]: Ah, so this is not referring to Core as in Self-Hosted CockroachDB, it is actually [Core changefeeds](https://www.cockroachlabs.com/docs/v23.1/changefeed-for), which are our implementation of changefeeds that customers do not need an Enterprise license for. Thanks for checking that though!

**Pull request merged**